compose-backstack/src/main/java/com/zachklipp/compose/backstack/BackstackFrame.kt

```diff
@@ -0,0 +1,44 @@
+package com.zachklipp.compose.backstack
+
+import androidx.compose.runtime.Composable
+
+/**
+ * Models a frame in a Backstack, with a unique [key] to identify it,
+ * and a [Content] function to display it.
```

Comment:
Nit: it would be helpful to explain to maybe what the key is used for so that people would have some idea of what sort of thing to pass in.